games-emulation/ppsspp/ppsspp-1.10.3.ebuild

```diff
@@ -1,59 +1,67 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit cmake-utils desktop
+inherit cmake desktop xdg
 
 DESCRIPTION="A PSP emulator written in C++"
 HOMEPAGE="https://www.ppsspp.org/"
 SRC_URI="
 	https://github.com/hrydgard/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
-	!system-ffmpeg? ( https://github.com/hrydgard/ppsspp-ffmpeg/archive/a2e98d7ba4c7c5cac08608732c3058cb46e3e0ef.tar.gz -> ${P}-ffmpeg.tar.gz )
-	https://github.com/hrydgard/ppsspp-lang/archive/1e3e4a0ba0ca8c0a092e027dfb7c1c4778366db5.tar.gz -> ${P}-assets_lang.tar.gz
-	https://github.com/hrydgard/pspautotests/archive/d02ba7407050f445edf9e908374ad4bf3b2f237b.tar.gz -> ${P}-pspautotests.tar.gz
-	https://github.com/hrydgard/minidx9/archive/7751cf73f5c06f1be21f5f31c3e2d9a7bacd3a93.tar.gz -> ${P}-dx9sdk.tar.gz
-	https://github.com/hrydgard/glslang/archive/2edde6665d9a56ead5ea0e55b4e64d9a803e6164.tar.gz -> ${P}-ext_glslang.tar.gz
-	https://github.com/Kingcom/armips/archive/8b4cadaf62d7de42d374056fc6aafc555f2bc7dc.tar.gz -> ${P}-ext_armips.tar.gz
-	https://github.com/Kingcom/tinyformat/archive/b7f5a22753c81d834ab5133d655f1fd525280765.tar.gz -> ${P}-ext_armips_ext_tinyformat.tar.gz
-	https://github.com/KhronosGroup/SPIRV-Cross/archive/90966d50f57608587bafd95b4e345b02b814754a.tar.gz -> ${P}-ext_SPIRV-Cross.tar.gz
+	https://github.com/KhronosGroup/SPIRV-Cross/archive/a1f7c8dc8ea2f94443951ee27003bffa562c1f13.tar.gz -> ${P}-ext_SPIRV-Cross.tar.gz
+	https://github.com/Kingcom/armips/archive/7885552b208493a6a0f21663770c446c3ba65576.tar.gz -> ${P}-ext_armips.tar.gz
+	https://github.com/Tencent/rapidjson/archive/73063f5002612c6bf64fe24f851cd5cc0d83eef9.tar.gz -> ${P}-ext_rapidjson.tar.gz
+	https://github.com/discordapp/discord-rpc/archive/3d3ae7129d17643bc706da0a2eea85aafd10ab3a.tar.gz -> ${P}-ext_discord-rpc.tar.gz
+	https://github.com/hrydgard/glslang/archive/d0850f875ec392a130ccf00018dab458b546f27c.tar.gz -> ${P}-ext_glslang.tar.gz
+	https://github.com/hrydgard/ppsspp-freetype/archive/cbea79dc8fef4d9210e2bac7e7b9b5ff3388197a.tar.gz -> ${P}-ext_native_tools_prebuilt.tar.gz
+	https://github.com/hrydgard/ppsspp-lang/archive/fd6f5bc01afb419eacca538b425c01eb2a055393.tar.gz -> ${P}-assets_lang.tar.gz
+	!system-ffmpeg? ( https://github.com/hrydgard/ppsspp-ffmpeg/archive/d3e695286cce346274192c214c00f021c00cd575.tar.gz -> ${P}-ffmpeg.tar.gz )
 "
 
+RESTRICT="test"
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
-IUSE="headless +qt5 sdl +system-ffmpeg"
+KEYWORDS="~amd64"
+IUSE="+headless qt5 sdl system-ffmpeg"
```

Comment:
What's the reason for flipping those defaults? Don't we want system libs used wherever possible? Could someone be surprised by `+headless` default?

I would expect an explanation inside git summary.

Reply:
@a17r upstream doesn't really support building with system ffmpeg.
As for headless I think it's a good addition that doesn't really require much (look at the deps).
My reasoning is similar to snes9x where (yes - its not a flag but) you (by default) get a headless version.

Reply:
I see it does not even cause any additional dependencies. If it does not cause a measurable uptick in build image size I'd argue it shouldn't even be an `IUSE`, but default ON certainly doesn't hurt here.

Re system ffmpeg, upstreams don't always share goals with distributions, doesn't mean we easily give in, unless the option is known to be problematic.

Reply:
> Re system ffmpeg, upstreams don't always share goals with distributions, doesn't mean we easily give in, unless the option is known to be problematic.

True, I myself run with all system-* on. Will switch to +.